main.go

```diff
@@ -0,0 +1,150 @@
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	"github.com/operator-framework/operator-lib/leader"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	hivev1 "github.com/openshift/hive/apis/hive/v1"
+	"github.com/openshift/operator-custom-metrics/pkg/metrics"
+
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	deadmanssnitchv1alpha1 "github.com/openshift/deadmanssnitch-operator/api/v1alpha1"
+	operatorconfig "github.com/openshift/deadmanssnitch-operator/config"
+	controllers "github.com/openshift/deadmanssnitch-operator/controllers/deadmanssnitchintegration"
+	"github.com/openshift/deadmanssnitch-operator/pkg/localmetrics"
+	//+kubebuilder:scaffold:imports
+)
+
+var (
+	scheme      = k8sruntime.NewScheme()
+	setupLog    = ctrl.Log.WithName("setup")
+	metricsPath = "/metrics"
+	// metricsPort the port on which metrics is hosted, don't pick one that's already used
+	metricsPort = "8081"
+)
+
+func init() {
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+
+	utilruntime.Must(hivev1.AddToScheme(scheme))
+	utilruntime.Must(deadmanssnitchv1alpha1.AddToScheme(scheme))
+	//+kubebuilder:scaffold:scheme
+}
+
+func printVersion() {
+	setupLog.Info(fmt.Sprintf("Go Version: %s", runtime.Version()))
+	setupLog.Info(fmt.Sprintf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH))
+}
+
+func main() {
+	var enableLeaderElection bool
+	var probeAddr string
+	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8080", "The address the probe endpoint binds to.")
+	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
+		"Enable leader election for controller manager. "+
+			"Enabling this will ensure there is only one active controller manager.")
+	opts := zap.Options{
+		Development: true,
+	}
+	opts.BindFlags(flag.CommandLine)
+	flag.Parse()
+
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
+
+	// Print configuration info
+	printVersion()
+	if err := operatorconfig.SetIsFedramp(); err != nil {
+		setupLog.Error(err, "failed to get fedramp value")
+		os.Exit(1)
+	}
+	if operatorconfig.IsFedramp() {
+		setupLog.Info("running in fedramp environment.")
+	}
+
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme:                 scheme,
+		MetricsBindAddress:     "0", // disable the controller-runtime metrics
+		Port:                   9443,
+		HealthProbeBindAddress: probeAddr,
+		LeaderElection:         enableLeaderElection,
+		LeaderElectionID:       "3a94b80a.deadmanssnitch.managed.openshift.io",
+	})
+	if err != nil {
+		setupLog.Error(err, "unable to start manager")
+		os.Exit(1)
+	}
+
+	err = leader.Become(context.TODO(), "memcached-operator-lock")
```

Comment:
It should use the same lock file name it did previously

Reply:
Thank you.
I have fixed that.